README.md

```diff
@@ -86,15 +64,15 @@ The `check` and `query` methods are overloaded for most standard JSON types, whi
 ### Result Types
 
 #### `OpaClient.check`
-For the `check` method, the output type is always `bool?`, allowing API users to disambiguate `undefined` from normal `true`/`false` results.
+For the `check` method, the output type is always `bool`.
 
-#### `OpaClient.query<T>`
-For the `query` method, the output type is configurable using generics, as shown in the example below.
+#### `OpaClient.evaluate<T>`
+For the `evaluate` method, the output type is configurable using generics, as shown in the example below.
 
 ```csharp
 string path = "authz/accounts/max_limit";
 
-double maxLimit = opa.query<double>("example", path);
+double maxLimit = opa.evaluate<double>(path, "example");
```

Comment:
`````suggestion
double? maxLimit = opa.evaluate<double>(path, "example");
```

For cases where a default value is appropriate, you can avoid the nullable `double` type in the example like so:
```csharp
string path = "authz/accounts/max_limit";

double maxLimit = opa.evaluate<double>(path, "example") ?? 0.0f;
```
`````